cargo

```diff
@@ -1,5 +1,9 @@
 #!/usr/bin/env bash
 
+
+echo "Notice: please run `cargo` directly. This cargo wrapper script will be removed in the future"
```

Comment:
I can't comment further down, but how about adding this line to the `stable` and `*` cases, and for the `nightly` case, add a line like:

```
echo "Notice: please use `./cargo-nightly`. This cargo wrapper script will be removed in the future"
```